openedx/features/course_experience/__init__.py

```diff
@@ -75,7 +75,16 @@
 # Waffle flag to enable relative dates for course content
 RELATIVE_DATES_FLAG = ExperimentWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'relative_dates', __name__, experiment_id=17)
 
-# Waffle flag to enable user calendar syncing
+# .. toggle_name: course_experience.calendar_sync
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: This course flag enables a course tool (which is a tool that is added on a course home page)
+#   that sends course assignment calendars to course students, whenever they click on the "Subscribe to calendar
+#   updates" button. The email contains an ics attachment that students can then use to sync with their own calendar
+#   apps. In order to enable this, the RELATIVE_DATES_FLAG must be enabled, too.
+# .. toggle_use_cases: open_edx
```

Comment:
@jmyatt: Is this meant to be a permanent toggle, or is it just needed for an extended rollout?
@regisb: We can land in a day or so, and update if we learn later that this should change.

Reply:
this toggle is for a feature that was shelved and will probably never be released, but will instead go on a list of tech debt / code to be removed.  Not sure if that provides any clarity on how to document this toggle at the moment.

Reply:
Yes it does.  Thank you @jmyatt.

Reply:
@regisb: Can you update to temporary and update the docs accordingly?

Reply:
If this feature is temporary, what is the target removal date?

Reply:
@regisb: I've been using the 3 month estimate from the documentation, as in 3 months from the creation date. If the date has passed already, that is fine.  It should bring this to someone's attention someday, and they can update the date or actually implement the removal. 😄

Reply:
Approved once this is resolved.  Thanks!

Reply:
:ok_hand: